support-frontend/assets/pages/[countryGroupId]/components/thankYouComponent.tsx

```diff
@@ -201,26 +199,49 @@ export function ThankYouComponent({
 	const isRegisteredAndSignedIn = !isNotRegistered && isSignedIn;
 	const isRegisteredAndNotSignedIn = !isNotRegistered && !isSignedIn;
 
-	let benefitsChecklist;
-	if (isTier) {
-		const productDescription = showNewspaperArchiveBenefit
-			? productCatalogDescriptionNewBenefits(countryGroupId)[productKey]
-			: productCatalogDescription[productKey];
-		benefitsChecklist = [
-			...productDescription.benefits
-				.filter((benefit) => filterBenefitByRegion(benefit, countryGroupId))
-				.map((benefit) => ({
-					isChecked: true,
-					text: benefit.copy,
-				})),
-			...(productDescription.benefitsAdditional ?? [])
-				.filter((benefit) => filterBenefitByRegion(benefit, countryGroupId))
-				.map((benefit) => ({
-					isChecked: true,
-					text: benefit.copy,
-				})),
-		];
-	}
+	const getBenefits = (): BenefitsCheckListData[] => {
```

Comment:
There's some logic later on that means we only display benefits for TierThree, but here we enable all 3 products in case that changes later